script/Deploy.s.sol

```diff
@@ -2,36 +2,53 @@
 
 pragma solidity 0.8.23;
 
-import { DeployBase } from "./DeployBase.s.sol";
+import { Script, console2 } from "../lib/forge-std/src/Script.sol";
 
-contract Deploy is DeployBase {
+import { IRegistrar } from "../src/interfaces/IRegistrar.sol";
+
+import { DeployBase } from "./DeployBase.sol";
+
+contract Deploy is Script, DeployBase {
     uint256 internal constant _STANDARD_PROPOSAL_FEE = 1e18; // 1 WETH
 
-    // NOTE: Ensure these are the correct cash token addresses.
+    // NOTE: Populate this arrays with cash token addresses.
     address[] internal _ALLOWED_CASH_TOKENS = [
         address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2) // mainnet WETH
```

Comment:
needs to include M

Reply:
This is handle in core, not sure we need this file anymore.

Reply:
Still good to keep this file to be able to deploy protocol and ttg standalone.